pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>com.hotels</groupId>
     <artifactId>hotels-oss-parent</artifactId>
-    <version>4.0.1</version>
+    <version>5.0.0</version>
```

Comment:
Add this change to the CHANGELOG and you'll also need to update the profile name in the travis.yml in the repo root.